app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php

```diff
@@ -638,6 +642,29 @@ public function resizeFile($source, $keepRatio = true)
         return false;
     }
 
+    /**
+     * Return width height for the image resizing.
+     *
+     * @param string $source
```

Comment:
Can you please define return param?

Reply:
but they already defined in strict types ? why need to duplicate it ?

Reply:
Agree. Despite its a requirement, i prefer to define a return type also, so for example 
@return string[] or int[]